CMakeLists.txt

```diff
@@ -21,6 +21,12 @@ if(NOT MSVC OR MSVC_VERSION LESS 1900)
   )
 endif()
 
+# Enable conformance mode for newer versions of MSVC
```

Comment:
`/d1parsePackExpressions-` does not enable conformance mode (under Project Settings -> C/C++ -> Language). It only affects the new code for fold expressions.

We should either update the comment or the option. It might make sense to use `/permissive-` if we can. We'd like to switch platform over to conformance mode soon and autowiring could be a good first step.